specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -26,6 +26,70 @@ These are the global settings for the Machine Learning Services API.
+      - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/connections/{connectionName}/raiBlocklists/{raiBlocklistName}"].put
+  - code: AvoidAdditionalProperties
+    reason: These schemas are already in production use,.
+    where:
+      - $.definitions.WorkspaceUpdateParameters.properties.properties.properties.managedNetwork.properties.outboundRules
+      - $.definitions.WorkspacePropertiesUpdateParameters.properties.managedNetwork.properties.outboundRules
+      - $.definitions.EndpointModels.properties.value.items.properties.capabilities
+      - $.definitions.AccountModel.properties.finetuneCapabilities
+      - $.definitions.AccountModel.properties.capabilities
+      - $.definitions.EndpointModels.properties.value.items.properties.finetuneCapabilities
+      - $.definitions.CustomKeysWorkspaceConnectionProperties.properties.credentials.properties.keys
+      - $.definitions.CustomKeys.properties.keys
+      - $.definitions.WorkspaceConnectionPropertiesV2.properties.metadata
+      - $.definitions.PATAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.SASAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.UsernamePasswordAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.NoneAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.ManagedIdentityAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.WorkspaceConnectionPropertiesV2BasicResource.properties.properties.properties.metadata
+      - $.definitions.WorkspaceConnectionPropertiesV2BasicResourceArmPaginatedResult.properties.value.items.properties.properties.properties.metadata
+      - $.definitions.AADAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.AccessKeyAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.AccountKeyAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.ApiKeyAuthWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.CustomKeysWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.OAuth2AuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+      - $.definitions.ServicePrincipalAuthTypeWorkspaceConnectionProperties.allOf[0].properties.metadata
+  - code: GuidUsage
+    reason: This property has always been a GUID, we just didn't mark its format before, this can't be change without breaking the customer.
+    where:
+      - $.definitions.WorkspaceConnectionOAuth2.properties.clientId.format
 tag: package-preview-2024-04
```

Comment:
remove this line

Reply:
@kazrael2119 - All the changes in this PR are just a copy of this PR : https://github.com/Azure/azure-rest-api-specs/pull/27841  where everything has been reviewed and approved.

Reply:
in https://github.com/Azure/azure-rest-api-specs/pull/27841， it doesn't have this line
![image](https://github.com/Azure/azure-rest-api-specs/assets/98569699/721aa565-985a-486d-80c2-13fc058d48f7)
 https://github.com/Azure/azure-rest-api-specs/pull/27841/files#diff-15f73b1309f178ea6025060b87c7438506ad7929f99ac109a152465f290d1fb3R92

Reply:
@kazrael2119 - Hi ZiWei, I should have mentioned that before, sorry my bad. So you are seeing that diff was because of a readme file merge conflict with main on the original PR https://github.com/Azure/azure-rest-api-specs/pull/27841 which was the reason we have created this copy PR. If we remove that line we will again run into same merge conflict as the original PR https://github.com/Azure/azure-rest-api-specs/pull/27841

![image](https://github.com/Azure/azure-rest-api-specs/assets/142325886/a1ba71df-f14a-4bb5-985b-a6cb1dc87c0f)

Reply:
@qiaozha - We have removed the incorrect tag as per the ask above. Could you please for approve for JS SDK?